src/components/shared/alert/Alert.css

```diff
@@ -0,0 +1,21 @@
+.css-1ytlwq5-MuiAlert-icon {
+    align-items: center;
+    justify-content: center;
+    margin: 20px;
+}
+
+.MuiAlert-message {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    margin: 0 20px;
+}
+
+.MuiAlert-message>img {
+    padding: 15px 0 0 100px;
+}
+
+.MuiPaper-root {
+    min-height: fit-content;
+    min-width: fit-content;
+}
```

Comment:
aceste style-uri nu tin de componenta de card, nu trebuie de inclus in acest PR.